src/components/ArrayFilter/ArrayFilter.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './style.css';
+
+const ArrayFilter = (props) => {
+  const { filterList } = props;
+
+  const handleChangeFilter = (type) => {
+    const buttons = document.querySelectorAll('.filter-item');
+    buttons.forEach(item => {
+      if(item.id === type){
+        item.classList.add('active');
+      } else {
+        item.classList.remove('active');
+      }
+    })
+    props.handleChangeFilter(type);
```

Comment:
We should consider naming the functions distinctively since the readability will be affected because of the confusion of `props.handleChangeFilter` and `handleChangeFilter`.